examples/inverse/plot_morph_stc_to_fsaverage_dipy.py

```diff
@@ -0,0 +1,231 @@
+"""
+==================================================================
+Demonstrate morphing subject source volume to fsaverage using dipy
+==================================================================
+
+This example demonstrates how to morph an individual subject source space to a
+common reference space. For this purpose dipy (http://nipy.org/dipy/) will be
```

Comment:
do a proper ReSTructuredText link here (look up docs on how or see how we do it elsewhere)